templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -10,14 +10,16 @@
     "\n",
```

Comment:
Should we update "Time to complete" to mention the specific hardware we've chosen (16 A10s)?

Reply:
I'll leave that for the doc team to figure out. My understanding is that we should just time the default assuming they will just click go.